pkg/sdk/testint/data_metric_function_references_gen_integration_test.go

```diff
@@ -0,0 +1,41 @@
+package testint
+
+import (
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_DataMetricFunctionReferences(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	t.Run("view domain", func(t *testing.T) {
+		functionId := sdk.NewSchemaObjectIdentifier("SNOWFLAKE", "CORE", "AVG")
+		statement := "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES"
+		view, viewCleanup := testClientHelper().View.CreateView(t, statement)
+		t.Cleanup(viewCleanup)
+
+		err := client.Views.Alter(ctx, sdk.NewAlterViewRequest(view.ID()).WithSetDataMetricSchedule(*sdk.NewViewSetDataMetricScheduleRequest("5 MINUTE")))
```

Comment:
can you please document in this test that after setting it to  minutes we get a corn response and link an issue describing it?

Reply:
Done - linked a ticket created by views team.